pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -1,5 +1,11 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ pkgs, lib, nodejs, stdenv, buildGoModule, fetchFromGitHub }:
```

Comment:
This shouldn't import `pkgs` -- import the packages you need individually.

Reply:
Are you sure? This is exactly how all other packages in nixpkgs that import `node-composition.nix` do it, as far as I could find. And `node-composition.nix` needs `pkgs`... I could change that, but it specifically says "Do not edit!". :)

Reply:
> This is exactly how all other packages in nixpkgs that import `node-composition.nix` do it, as far as I could find.

It doesn't look like they use anything from `pkgs` directly, they only use them to import `node-composition.nix` -- if they are doing that, then that should be changed. (Which packages do that?)

To clarify: you use `pkgs` in `nativeBuildInputs` -- you should import those separately, at least.

> And node-composition.nix needs pkgs... I could change that, but it specifically says "Do not edit!". :)

My bad! Yes, keep it, but only for that :)

Reply:
Ah, yes, you're right. Most of them import each input separately. Not all though, like `matrix-appservice-slack` or `spacegun`.

Reply:
Should be okay now.